### PR TITLE
Wrapup post for 11/13/2015

### PR DESCRIPTION
**Pull request by rgbkrk**
Ready for review.

### CODE REVIEW

**Comment by carolynvs** [comment]
Looks like the title is out of date? I don't see any mention of jobs.

**Comment by rgbkrk** [comment]
@carolynvs thanks! Also fixed the path to the 2FA gif.

**Comment by carolynvs** [comment]
:+1:

**Comment by rgbkrk** [comment]
Updated. As you @ktbartholomew. :wink:

File: _posts/2015-11-13-weekly-news.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: "Weekly News, DVM, carina updates"
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: Capitalize "Carina Updates"

File: _posts/2015-11-13-weekly-news.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  In this week's roundup: Docker 1.9 and Swarm 1.0 now on all Carina clusters, updates for the Carina CLI and GUI, official dvm release!
+categories:
+ - Docker
+ - Swarm
+ - Carina
+ - News
+authorIsRacker: true
+---
+
+## Docker 1.9 and Swarm 1.0 Available on all new clusters
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: Docker 1.9 and Swarm 1.0 are available on all new clusters

File: _posts/2015-11-13-weekly-news.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+Launch a cluster, get Docker 1.9 with Swarm 1.0. The biggest feature here is the new `docker network` command. Right now you can use isolated networks per swarm host; overlay networking is not available yet.
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: Launch a cluster and get ...

"biggest feature" > "most important feature" ?

... per Swarm host ...

File: _posts/2015-11-13-weekly-news.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Check out [Docker 1.9's release notes](https://github.com/docker/docker/releases/tag/v1.9.0) for more details on new `docker` options and [Swarm 1.0's release notes](https://github.com/docker/swarm/releases/tag/v1.0.0) for even more.
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: Read the [Docker 1.9 release notes](https://github.com/docker/docker/releases/tag/v1.9.0) for more details about new `docker` options and the [Swarm 1.0 release notes](https://github.com/docker/swarm/releases/tag/v1.0.0) for even more.

File: _posts/2015-11-13-weekly-news.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+## Docker Version Manager Release
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: release

File: _posts/2015-11-13-weekly-news.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+We're proud to launch a cross platform (Windows, OS X, Linux) and multi-shell (bash, zsh, csh, powershell, cmd) compatible release of `dvm`, the Docker Version Manager.
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: cross-platform

(Bash, zsh, csh, PowerShell, CMD)

File: _posts/2015-11-13-weekly-news.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+This lets you swap between docker versions with a simple shell command, `dvm use`:
+
+```bash
+$ dvm use 1.8.2
+Now using Docker 1.8.2
+$ dvm use 1.9.0
+Now using Docker 1.9.0
+```
+
+Installing new Docker versions is simple enough:
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: ... is as simple as:

File: _posts/2015-11-13-weekly-news.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+```
+$ dvm install 1.8.3
+Installing 1.8.3...
+Now using Docker 1.8.3
+```
+
+As well as listing the installed versions:
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: So is listing ...

File: _posts/2015-11-13-weekly-news.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+```
+$ dvm ls
+	1.8.2
+->	1.8.3
+	1.9.0
+	experimental (1.10.0-dev, build 0cdc96c)
+	system (1.9.0)
+```
+
+Be sure to check out the [DVM announcement](https://getcarina.com/blog/docker-version-manager/) and tutorial on [Managing Docker client versions with dvm](https://getcarina.com/docs/tutorials/docker-version-manager/) for more!
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: See the [DVM announcement](https://getcarina.com/blog/docker-version-manager/) ...

File: _posts/2015-11-13-weekly-news.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+## app.getcarina.com updates
+
+Existing Rackspace users may now log in to app.getcarina.com using multi-factor authentication. Multi-factor settings can be edited from the Rackspace Control Panel. Multi-factor authentication is not yet available for users who signed up for Carina without an existing Rackspace cloud account.
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: ... users can now log in to ...
... by using multi-factor authentication. You can edit multi-factor settings from the ...

File: _posts/2015-11-13-weekly-news.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+
+[2fa is the way]({% asset_path 2015-11-13-weekly-news/carina-2fa.gif %})
+
+## Carina CLI update
+
+### Carina on Homebrew
+
+Carina is now available through [Homebrew on OS X](http://brew.sh/) for installation:
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: You can now install Carina through [Homebrew on OS X](http://brew.sh/):

File: _posts/2015-11-13-weekly-news.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+
+```bash
+brew install carina
+```
+
+If you were using it before with a direct download, you may want to delete that binary and update any `PATH`s you were setting before. Now a package manager has your back. When an update to the carina CLI is available, run
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: Consider the following revision:

If you were using Carina before with a direct download, consider deleting that binary, updating any PATH variables that you previously set, and using the package manager. If you do, when an update to the `carina` CLI is available, you can simply run:

File: _posts/2015-11-13-weekly-news.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+### Notes on Carina
+
+* If you have `GITHUB_TOKEN` set, `carina` will rely on that when checking for updates from GitHub releases.
+* Tokens for Carina are cached in your `carina` config folder (OS X uses `~/.carina/cache.json`). Turn this off using `--no-cache`.
+
+### Ch-ch-ch-changes
+
+* The `--wait` flag has been fixed for `create` and `rebuild` (intermittent golang panics prior).
```
Comment (kmsholcomb) [inline _posts/2015-11-13-weekly-news.md]: "(intermittent golang panics prior)" > ? Is this parenthetical necessary? If so, could it be stated in a plainer and clearer way?

**Comment by kmsholcomb** [comment]
Hey @rgbkrk: I left a few copy edits in comments in the file. Take 'em or leave 'em!

**Comment by rgbkrk** [comment]
Taking a bunch of them now!

**Comment by rgbkrk** [comment]
Thanks @kmsholcomb!

**Comment by rgbkrk** [comment]
Gah, missed one.

**Comment by mendeni** [comment]
:+1:

**Pull request merged**